src/app/component-library/catalog/contract-offer-mini-list/contract-offer-mini-list.component.ts

```diff
@@ -7,14 +7,20 @@ import {
 } from '@angular/core';
 import {ContractNegotiationService} from '../../../core/services/contract-negotiation.service';
 import {ContractOffer} from '../../../core/services/models/contract-offer';
+import {ConnectorEndpointService} from '../connector-endpoint.service';
+
+export interface ContractOffersWithEndpoint {
+  contractOffers: ContractOffer[];
+  endpoint: string;
+}
 
 @Component({
   selector: 'contract-offer-mini-list',
   templateUrl: 'contract-offer-mini-list.component.html',
 })
 export class ContractOfferMiniListComponent {
   @Input()
-  contractOffers!: ContractOffer[];
```

Comment:
I believe the "Contract Offer" is an UI-internal model. Meaning it's not an API model. Meaning we can add fields to it. Can't we weave the endpoints into that object? I ask, because we do a mapping API Contract Offer -> UI Contract Offer anyways already.

Idk this case: Is it worth it?